src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -64,7 +64,7 @@
     "@types/file-saver": "2.0.1",
     "async-mutex": "0.2.4",
     "bootstrap": "5.2.3",
-    "carbon-components-angular": "5.25.1",
+    "carbon-components-angular": "5.26.0",
```

Comment:
let's add description in the commit on why this is being updated so that other people know the need for this update. This way everyone can follow and understand the changes. We should always focus on adding more information in the commit description whenever it is required.

Reply:
did  version changes for password input field it will support upper version

Reply:
Could you please add this in commit description as suggested by Nizam?